nvm.sh

```diff
@@ -2405,8 +2406,17 @@ nvm() {
       nvm_err "uname -a: '$(uname -a | awk '{$2=""; print}' | xargs)'"
       if [ "$(nvm_get_os)" = "darwin" ] && nvm_has sw_vers; then
         nvm_err "OS version: $(sw_vers | command awk '{print $2}' | command xargs)"
-      elif [ -r "/etc/issue" ]; then
```

Comment:
this seems like it could stay an `elif`, and the other logic added in an `else`?

Reply:
Nope, as we need to confirm the content of it first